lib/segment/src/types.rs

```diff
@@ -322,7 +322,8 @@ pub struct SearchParams {
     pub hnsw_ef: Option<usize>,
 
     /// Search without approximation. If set to true, search may run long but with exact results.
-    #[serde(default)]
+    /// For sparse vectors this option ignores inverted index and performs full-scan search.
+    #[serde(default, alias = "plain")]
```

Comment:
By this flag, I force plain search instead of searching in inverted index. Both of them are exact search, so that `exact` is not a good name for sparse vectors. So, I added alias `plain` because `plain` describes skipping in HNSW and inverted index both.

Reply:
the sparse search is always exact :)

Reply:
Right, that's why I added alias. Because this flag forces plain search in HNSW. So we can use alias and force plain search in sparse vectors with this flag too

Reply:
Will ask @generall. Because it's untrivial API change

Reply:
BTW I use this flag for benches